ee_core/include/ee_core.h

```diff
@@ -40,6 +40,9 @@ extern int iop_reboot_count;
 
 extern int padOpen_hooked;
 
+extern int enforceLanguage;
+extern int LanguageToEnforce;
```

Comment:
‘LanguageToEnforce’ Is this used?

Reply:
Yes, it's passed to the EE Core
If value is 1, configparam getter/setter get hooked